src/control-flow-basics/if.md

```diff
@@ -38,8 +38,8 @@ Because `if` is an expression and must have a particular type, both of its
 branch blocks must have the same type. Show what happens if you add `;` after
 `"small"` in the second example.
 
-When `if` is used in an expression, the expression must have a `;` to separate
-it from the next statement. Remove the `;` before `println!` to see the compiler
-error.
+`if` expression should be used in the same way as the other expressions. For
```

Comment:
I dislike starting sentences with code, so I would suggest adding a little word here (I hope it fits):

```suggestion
An `if` expression should be used in the same way as the other expressions. For
```